pkg/api/api.go

```diff
@@ -500,17 +506,19 @@ var FuncMap = template.FuncMap{
 }
 
 // NewScalewayAPI creates a ready-to-use ScalewayAPI client
-func NewScalewayAPI(endpoint, organization, token string) (*ScalewayAPI, error) {
+func NewScalewayAPI(apiEndPoint, accountEndPoint, organization, token string) (*ScalewayAPI, error) {
 	cache, err := NewScalewayCache()
 	if err != nil {
 		return nil, err
 	}
 	s := &ScalewayAPI{
-		APIEndPoint:  endpoint,
-		Organization: organization,
-		Token:        token,
-		Cache:        cache,
-		anonuuid:     *anonuuid.New(),
+		APIEndPoint:     apiEndPoint,
+		ACCOUNTEndPoint: accountEndPoint,
+		APIurl:          apiEndPoint,
```

Comment:
Ok, got it, you can keep this hack to keep only one instance of each functions :)